index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="#" method="POST">
```

Comment:
- [ ] Your form does not employ the [formspree service](https://formspree.io/html)  Let's kindly add it by editing the `action` attribute.  That way, your users will be able to communicate with you 💪